src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/StageDetail.component.js

```diff
@@ -0,0 +1,188 @@
+// @flow
+import React, { type ComponentType, useState } from 'react';
+import { withStyles } from '@material-ui/core';
+import i18n from '@dhis2/d2-i18n';
+import { colors,
+    spacersNum,
+    DataTableBody,
+    DataTableHead,
+    DataTableFoot,
+    DataTable,
+    DataTableRow,
+    DataTableCell,
+    DataTableColumnHeader,
+    Button,
+} from '@dhis2/ui';
+import { sortDataFromEvent } from '../hooks/sortFuntions';
+import { useComputeDataFromEvent, useComputeHeaderColumn } from '../hooks/useEventList';
```

Comment:
Can we move these files closer to where they are used? Meaning, since these files are only used in this file I would move the files to the StageDetail folder.

Reply:
the helpers also used in `Stage`, but as you suggested, I move all folder to `StageDetail`